packages/client/modules/userDashboard/components/OrgTeams/OrgTeamsRow.tsx

```diff
@@ -24,33 +27,70 @@ const OrgTeamsRow = (props: Props) => {
           isOrgAdmin
           isSelf
           email
+          preferredName
         }
       }
     `,
     teamRef
   )
   const {id: teamId, teamMembers, name} = team
   const teamMembersCount = teamMembers.length
+  const viewerTeamMember = teamMembers.find((m) => m.isSelf)
+  if (!viewerTeamMember && !isOrgAdmin) return null
+  const {isLead: viewerIsLead} = viewerTeamMember ?? {isLead: false}
+  const teamLead = teamMembers.find((m) => m.isLead)
+  const teamLeadName = teamLead && !viewerTeamMember?.isLead ? teamLead.preferredName : null
+  const {togglePortal, modalPortal} = useModal()
 
   return (
-    <Link
-      className='block hover:bg-slate-100 focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-inset'
-      to={`teams/${teamId}`}
-    >
-      <div className='flex items-center p-4'>
-        <div className='flex flex-1 flex-col py-1'>
-          <div className='text-gray-700 text-lg font-bold'>{name}</div>
-          <div className='flex items-center justify-between'>
-            <div className='text-gray-600'>
-              {`${teamMembersCount} ${plural(teamMembersCount, 'member')}`}
-            </div>
+    <>
+      <div className='flex items-center justify-between p-4 hover:bg-slate-100'>
+        <div className='flex flex-col'>
+          <div className='inline-flex items-center items-center gap-x-2'>
+            <Link to={`teams/${teamId}`} className='inline-block text-lg font-bold text-sky-500'>
+              {name}
+            </Link>
+            {viewerIsLead && (
+              <span className='rounded-full bg-sky-700 px-2 py-0.5 text-xs text-white'>
+                Lead team
+              </span>
+            )}
+            {!viewerIsLead && !isOrgAdmin && (
+              <span className='rounded-full bg-grass-500 px-2 py-0.5 text-xs text-white'>
+                Member team
+              </span>
+            )}
+          </div>
+          <div className='text-gray-600'>
+            {`${teamMembersCount} ${plural(teamMembersCount, 'member')}`}
           </div>
         </div>
-        <div className='flex items-center justify-center'>
-          <ChevronRight />
-        </div>
+        {(viewerIsLead || isOrgAdmin) && (
+          <div className='flex items-center'>
+            <div>
+              <LinkButton
+                aria-label='Click to permanently delete this team.'
+                palette='red'
+                onClick={togglePortal}
+              >
+                <IconLabel icon='remove_circle' label='Delete Team' />
+              </LinkButton>
+              <span className='mt-2 block text-sm text-slate-600'>
+                <b>Note</b>: {'This can’t be undone.'}
```

Comment:
+1 This clutters the view quite a bit, maybe we could just rename the button to "Delete permanently" as it can be inferred from the context that it means the team?